packages/tupaia-web/src/features/DashboardItem/EnlargedDashboardItem.tsx

```diff
@@ -179,19 +177,20 @@ export const EnlargedDashboardItem = () => {
     setUrlSearchParams(urlSearchParams);
   };
 
-  const titleText = `${currentDashboardItem?.name}, ${
-    currentDashboardItem?.entityHeader || activeDashboard?.entityName
+  const titleText = `${currentDashboardItem?.config?.name}, ${
+    // @ts-ignore - entityHeader is all lowercase in the types config
+    currentDashboardItem?.config?.entityHeader || entityName
```

Comment:
I think in theory we shouldn't need to do this, if it gets overwritten in the `type` of `DashboardItem`?